ophyd/controls/area_detector.py

```diff
@@ -228,7 +231,7 @@ def _set_shutter(self, value):
     def _start_acquire(self, **kwargs):
         """Do an actual acquisiiton"""
         if self._acq_count < self._acq_num:
-            self._set_shutter(self._acq_count % 2)
+            self._set_shutter(self._acq_count % 2 + 1)
```

Comment:
make sure this works at XPD and sort out a way to make this configurable so we do not break other beamlines.